infrastructure/variables.tf

```diff
@@ -44,6 +44,10 @@ variable "feature_certificateOfService" {
   default = "false"
 }
 
+variable "feature_inversionOfControl" {
```

Comment:
this mixed camel / snake case is triggering my _eyetwitch_. I know it's not your fault as there's already `certificateOfService` breaking this convention, should we change that too?

Reply:
Agree, but trying to make it consistent with rest of the flags. This is temporary flag which we will remove after go live.